courses/machine_learning/deepdive2/image_classification/solutions/3_tf_hub_transfer_learning.ipynb

```diff
@@ -53,6 +53,8 @@
      "name": "stdout",
```

Comment:
<p>nit: You could use a path join or something else so that you can get the URL not to go past the 80 character limit. I don't care either way.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>